### DISCUSSION TITLE
not installed in windows 11

### DISCUSSION THREAD

**Discussion by R43d-SEC** [projectdiscovery/nuclei#3494] [Q&A]
runtime/cgo
# runtime/cgo
gcc_libinit_windows.c: In function '_cgo_beginthread':
gcc_libinit_windows.c:136:27: error: implicit declaration of function '_beginthread'; did you mean '_cgo_beginthread'? [-Werror=implicit-function-declaration]
  136 |                 thandle = _beginthread(func, 0, arg);
      |                           ^~~~~~~~~~~~
      |                           _cgo_beginthread
cc1: all warnings being treated as errors

**Comment by ehsandeep** [comment]
@watskenet please include all the details, including cmd you used to run with complete error log.

**Comment by olearycrew** [comment]
@watskenet please provide the additional requested details so that we can help you with this issue. If we don't hear back from you soon, we'll assume this discussion is closed.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
